src/movie/movie.utils.ts

```diff
@@ -1,14 +1,41 @@
 import { Movie } from 'src/graphql'
 
-export const extractDataFromMetadataResponse = (response): Movie => {
-  const movie = response.data[0]
+export const buildPosterServiceURL = (imdbId: string): string => {
+  return `${process.env.POSTER_SERVICE_URL}?id=${imdbId}`
+}
 
-  return {
-    imdbId: movie.imdb_id,
-    title: movie.title,
-    releaseYear: parseInt(movie.release_year),
-    posterPath: '',
+export const buildMetadataServiceURL = (
```

Comment:
Maybe we could use a URL query builder here to make it more extendable with different query params. Will need to look at such libraries.

Reply:
Check it out: https://nodejs.org/docs/latest-v12.x/api/url.html

Reply:
const newSearchParams = new URLSearchParams(myURL.searchParams);
// The above is equivalent to
// const newSearchParams = new URLSearchParams(myURL.search);

newSearchParams.append('a', 'c');
console.log(myURL.href);
// Prints https://example.org/?a=b
console.log(newSearchParams.toString());
// Prints a=b&a=c

Reply:
Nice! A few weeks ago, I've read a medium article about that but forgot about it.